test/test_xcpp_kernel.py

```diff
@@ -133,6 +135,24 @@ class XCppTests(jupyter_kernel_test.KernelTests):
         ]
 
 
+def test_notebook():
+    # Execute the notebook
+    executed_notebook = pm.execute_notebook(
+        'Notebooks/Untitled.ipynb',
+        'Notebooks/Untitled_output.ipynb',
+        log_output=True,
+        kernel_name='xcpp20'
+    )
+
+    # Read the executed notebook
+    with open('Notebooks/Untitled_output.ipynb') as f:
+        nb = nbformat.read(f, as_version=4)
+
+    # Get the output of the third cell
+    output = nb.cells[2].outputs[0]['text']
+
+    assert output == '12'
```

Comment:
Where does that 12 come from?

Reply:
In the notebook I had `cout<<12;` So I'm checking here if the 12 has actually been printed.

Reply:
We should take it from the notebook itself rather than hardcoding it.

Reply:
Yes, I take it from the notebook and check if the value is true. Line 148 to 152 is reading the notebook.

Reply:
I understand that but we should read the value of 12 from there too.

Reply:
I guess I am confused. Do we run the notebook somewhere?